### PR TITLE
Misc changes and fixes

### PR DESCRIPTION
**Pull request by marvis999**

### CODE REVIEW

File: Vindicta.Altis/UI/BuildUI/BuildUI.sqf [hunk 1]
```diff
@@ -253,7 +253,7 @@ CLASS("BuildUI", "")
 				(_display displayCtrl IDC_TOOLTIP1) ctrlsetStructuredText parseText format ["<t color='%1' align='center' valign='bottom'>TAB:</t> <t color='%1' align='center' valign='bottom' font='RobotoCondensedLight'> BUILD/PICK UP/DROP OBJECTS</t>", _color];
 				(_display displayCtrl IDC_TOOLTIP2) ctrlsetStructuredText parseText format ["<t color='%1' align='center' valign='bottom'>BACKSPACE: </t> <t color='%1' align='center' valign='bottom' font='RobotoCondensedLight'> CLOSE MENU</t> <t color='%1' align='center' valign='bottom'>  |  ARROW KEYS: </t> <t color='%1' align='center' valign='bottom' font='RobotoCondensedLight'> NAVIGATE MENU</t>", _color];
 
-				(_display displayCtrl IDC_ITEXTBG) ctrlSetBackgroundColor [0,0,0,0];
+				//(_display displayCtrl IDC_ITEXTBG) ctrlSetBackgroundColor [0,0,0,0];
```
Comment (marvis999) [inline Vindicta.Altis/UI/BuildUI/BuildUI.sqf]: disables hint text in the UI blinking evry time category is switched

File: Vindicta.Altis/UI/InGameMenu/InGameMenu.sqf [hunk 2]
```diff
@@ -31,7 +31,7 @@ CLASS("InGameMenu", "DialogBase")
 
 			pr _text = format ["Mission Menu  v%1", call misc_fnc_getVersion];
 			T_CALLM1("setHeadlineText", _text);
-			T_CALLM1("setHintText", "VINDCTA pre-alpha version");
+			T_CALLM1("setHintText", "VINDICTA pre-alpha version.");
```
Comment (marvis999) [inline Vindicta.Altis/UI/InGameMenu/InGameMenu.sqf]: oops

File: Vindicta.Altis/UI/Resources/MissionUIControlClasses.h [hunk 3]
```diff
@@ -144,9 +144,9 @@ class MUI_BUTTON_TXT : RscButton
 
 	colorBackground[] = MUIC_BLACK;
 	colorBackgroundActive[] = MUIC_WHITE;
-	colorBackgroundDisabled[] = MUIC_BLACK;
+	colorBackgroundDisabled[] = MUIC_TXT_DISABLED;
```
Comment (marvis999) [inline Vindicta.Altis/UI/Resources/MissionUIControlClasses.h]: fixed disabled buttons having invisible text

File: Vindicta.Altis/Undercover/UndercoverMonitor.hpp [hunk 4]
```diff
@@ -28,7 +28,7 @@
 
 // suspicion values for looking into inventory
 #define SUSP_INV_CIV 0.2							// Looking into civilian vehicle or own inventory
-#define SUSP_INV_MIL 0.4							// Looking into military inventory
```
Comment (marvis999) [inline Vindicta.Altis/Undercover/UndercoverMonitor.hpp]: bit more suspicion for accessing inventories... a lot more

File: Vindicta.Altis/Undercover/undercoverMonitor.sqf [hunk 5]
```diff
@@ -108,6 +108,9 @@ CLASS("UndercoverMonitor", "MessageReceiver");
 		_unit setVariable [UNDERCOVER_WANTED, false, true];					// GLOBAL: if true player unit is hostile and "setCaptive false"
 		_unit setVariable [UNDERCOVER_SUSPICIOUS, false, true];				// GLOBAL: true if player is suspicious (suspicion variable >= SUSPICIOUS #define)													
 
+		// make everyone ACE medic, temp fix I guess
```
Comment (marvis999) [inline Vindicta.Altis/Undercover/undercoverMonitor.sqf]: everyone is now an ACE medic.. yay

File: Vindicta.Altis/Undercover/undercoverMonitor.sqf [hunk 6]
```diff
@@ -577,10 +580,16 @@ CLASS("UndercoverMonitor", "MessageReceiver");
...
 
 						if (activeACE) then { 
-							if !(_unit getVariable ["ACE_isUnconscious", false]) then { T_CALLM("setState", [sUNDERCOVER]); };
+							if !(_unit getVariable ["ACE_isUnconscious", false]) then { 
+								T_CALLM("setState", [sUNDERCOVER]); 
+								OOP_INFO_0("ACE_isUnconscious is false, leaving state sINCAPACITATED");
+							};
 						};
 
-						if (lifeState _unit == "HEALTHY") then { T_CALLM("setState", [sUNDERCOVER]); };
```
Comment (marvis999) [inline Vindicta.Altis/Undercover/undercoverMonitor.sqf]: fixed problem where UM state continuously switches between overt and undercover while unconscious

File: Vindicta.Altis/Undercover/UndercoverMonitor.hpp [hunk 7]
```diff
@@ -28,7 +28,7 @@
...
+#define SUSP_INV_MIL 0.9							// Looking into military inventory
```
Comment (Sparker95) [inline Vindicta.Altis/Undercover/UndercoverMonitor.hpp]: I think you don't know how it works fully.
Each time you grab an item from inventory, suspiciousness boosts by a large amount for the duration of next cycle (1 second). So you can look inside inventory without being shot at (although bots will try to arrest you if they see that), but grabbing something from inventory makes you shot, of course if they see that. Now you make people instantly shot if they look into military vehicles. I don't think that it's fair.

**Pull request merged**